framework/rbac/ManagerInterface.php

```diff
@@ -180,18 +180,18 @@ public function hasChild($parent, $child);
     public function getChildren($name);
 
     /**
-     * Assigns a role to a user.
+     * Assigns a role or permission to a user.
```

Comment:
Permission wasn't documented on purpose since it's a bad practice to assign permissions to users directly.